SYCL/Reduction/reduction_nd_s0_dw.cpp

```diff
@@ -1,5 +1,4 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
-// RUNx: %HOST_RUN_PLACEHOLDER %t.out
```

Comment:
Is this feature device-only? Or are we going to add support for HOST in the future? If the second, probably it makes sense to keep this line or add todo to not forget to enable it.

Reply:
running on host would require adding support for barrier() and group algorithm reduce(). That is a known issue that cannot be forgotten. Also several other tests already mention that (e.g. reduction_placeholder.cpp, reduction_queue_parallel_for.cpp )

Reply:
I can return that line if you think it is useful to have it there.